source/manual/dns.html.md

```diff
@@ -34,135 +34,12 @@ Note that we __do not__ manage any other DNS records: if you get a request conce
-
-    ```
-    # google_dns_record_set.NS_passport will be updated in-place
-    ~ resource "google_dns_record_set" "NS_passport" {
-          id           = "10"
-          name         = "passport.service.gov.uk."
-        ~ rrdatas      = [
-            - "ns1.p08.dynect.net.",
-            - "ns2.p08.dynect.net.",
-            - "ns3.p08.dynect.net.",
-            - "ns4.p08.dynect.net.",
-            + "ns-144.awsdns-18.com.",
-            + "ns-1559.awsdns-02.co.uk.",
-            + "ns-1435.awsdns-51.org.",
-            + "ns-535.awsdns-02.net.",
-          ]
-          # (4 unchanged attributes hidden)
-      }
-    ```
-
-    > It's normal to see changes in TXT records relating to escaping of quotes. You can safely ignore these if they don't change any of the content of the record. This is a bug in the way we handle splitting long TXT records between AWS and GCP in our [YAML -> Ruby -> Terraform process](https://github.com/alphagov/govuk-dns).
-
-1. Apply the change in AWS (i.e. save the changes):
-
-    ```
-    gds govuk dns --provider aws --zone service.gov.uk --action apply --role govuk-production-poweruser
-    ```
-
-1. Apply the change in GCP (i.e. save the changes):
-
-    ```
-    gds govuk dns --provider gcp --zone service.gov.uk --action apply --role govuk-production-poweruser
-    ```
-
-    > Sometimes, the GCP deployment requires multiple runs. This is because, in order to change a DNS record, the Google provider deletes and re-adds that record. This can cause a [race condition](https://github.com/alphagov/govuk-dns/issues/67) where Google tries to create the new one before it has successfully deleted the old one. In this case, the build will fail, and you just need to re-run the GCP `apply` job.
-
-1. Revoke the GitHub personal access token by navigating to the [personal access token](https://github.com/settings/tokens) page and clicking 'Delete'.
-
-### Running govuk-dns manually using Jenkins
-
-If you run into issues using the `gds govuk dns` command, you can trigger a build of the [Deploy_DNS Jenkins job][] directly.
-
-You will need AWS credentials whichever provider you're targetting. You can get these by running:
-
-```
-gds aws govuk-production-poweruser -e
-```
-
-To plan or apply using the `gcp` provider, you will also need a Google OAuth access token. Once you [have gcloud set up][Google Cloud Platform (GCP)] you can get these by running:
-
-```
-gcloud config set project govuk-production && gcloud auth login --brief && gcloud auth print-access-token
-```
-
-You'll be prompted to login to your Google account to allow Google Cloud SDK access your Google Account.
+1. Ensure you have [Terraform Cloud access](#getting-terraform-cloud-access)
```

Comment:
Would be good to have a link to terraform cloud itself here (e.g. I've got access already, but can't remember the URL)